content/news/playtest-20160904.md

```diff
@@ -0,0 +1,68 @@
+
+<div style="text-align:center" markdown="1">
+![New content installer](/images/news/20160904-cnc-contentinstall.png)
+</div>
+
+Because we have changed which assets are copied, all players will need to reinstall (or redownload) the assets for this playtest. These will be used for the future playtests and releases, so only needs to be done once.
+
+The new FMV installation support has arrived just in time (some would argue far too late!), because this playtest includes 7 more campaign missions for TD and one new campaign mission each for RA and D2K. RA also includes three custom multiplayer missions that can be selected from the mission chooser (for single-player) or the multiplayer lobby (for cooperative play).
+
+Other notable changes in this release include:
+
+* Improvements to unit pathfinding behaviour (less traffic jams around bridges and choke points!)
+* Aircraft in RA will now intelligently manage their reloading behaviour at airfields or helipads
+* New hotkeys and mouse bindings for moving around the map and producing units
+* Lobby options (such as crates and shroud) are no longer reset when switching maps
+* Improved UPnP support for automatically opening ports when hosting local servers
+* Fixes for a collection of common crashes that sneaked in to the last release (we're sorry about that!)
+* A collection of balance changes for RA and TD that were discussed and tested on our [community forum](http://www.sleipnirstuff.com/forum/viewforum.php?f=82)
+* Many trait and Lua API improvements for mods and maps
+
+<div style="text-align:center" markdown="1">
+![AI Engineers](/images/news/20160904-ra-aiengineer.png)
+
+Watch your back! We have taught the AI to use Engineers to capture tech buildings and enemy structures.
+<br />
+A controverial exploit allowing engineers to cancel production in the RA mod has also been fixed.
+</div>
+
+
+<div style="text-align:center" markdown="1">
+![Player scores](/images/news/20160904-d2k-playerscore.png)
+
+A new scoring system tracks many ingame actions, not just kills.
+<br />
+The redesigned score menu now also lists the spectators that are observing the match.
+</div>
+
+<div style="text-align:center" markdown="1">
+![Burning trees](/images/news/20160904-cnc-burningtrees.png)
+
+We have restored the ability for flame weapons to burn trees down to charred husks.
+<br />
+Try the playtest today to find all the other cosmetic improvements!
+</div>
+
+Check out [the full changelog](http://changelog.openra.net) for more details, and head over to the [download page](/download/) to try it for yourself!
```

Comment:
Here's the changelog permalink: https://github.com/OpenRA/OpenRA/wiki/Changelog/c25c6528e4eeca1e9e36c2b25c2fbae8b4d9c363